docs/ref/models/querysets.txt

```diff
@@ -3914,7 +3914,7 @@ of the return value
 ``filter``
 ~~~~~~~~~~
 
-An optional :class:`Q object <django.db.models.Q>` that's used to filter the
+An optional :class:`Q <django.db.models.Q>` object that's used to filter the
```

Comment:
Seems like it could be simplified further as follows?

```suggestion
An optional :class:`~django.db.models.Q` object that's used to filter the
```